bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx81.html

```diff
@@ -28,32 +28,30 @@
 <main class="content-wrapper mzp-t-firefox">
   {% include 'firefox/whatsnew/includes/header.html' %}
 
-  <section class="content-main">
+  <section class="wnp-content-main">
     <div class="mzp-l-content mzp-t-content-md">
-      <div class="mzp-c-emphasis-box">
-        <img class="c-emphasis-box-logo" src="{{ static('img/firefox/whatsnew/whatsnew81/hero-device.png') }}" width="95" alt="">
-        <h1 class="c-emphasis-box-title">{{ ftl('whatsnew81-main-title') }}</h1>
-        <p class="main-tagline">{{ ftl('whatsnew81-main-intro') }}</p>
-
-      {% if show_send_to_device %}
-        <div id="send-to-device-wrapper" class="primary-cta">
-        {{ send_to_device(include_title=False, message_set='fx-whatsnew', input_label=ftl('whatsnew81-send-form-label'), input_label_alt=ftl('whatsnew81-send-form-label-alt'), class='vertical') }}
-        </div>
-      {% else %}
-        <div class="qr-code-wrapper">
-          <img src="{{ static('img/firefox/mobile/protocol/qr-firefox.png') }}"
-              width="220"
-              height="220"
-              id="firefox-qr"
-              data-mozillaonline-link="{{ static('img/firefox/mobile/protocol/qr-firefox-mozillaonline.png') }}"
-              alt="{{ ftl('whatsnew81-scan-this-qr-code') }}">
-        </div>
-      {% endif %}
+      <img class="wnp-main-image" src="{{ static('img/firefox/whatsnew/whatsnew81/hero-device.png') }}" width="95" alt="">
+      <h1 class="wnp-main-title">{{ ftl('whatsnew81-main-title') }}</h1>
+      <p class="wnp-main-tagline">{{ ftl('whatsnew81-main-intro') }}</p>
+
+    {% if show_send_to_device %}
+      <div id="send-to-device-wrapper" class="primary-cta">
+      {{ send_to_device(include_title=False, message_set='fx-whatsnew', input_label=ftl('whatsnew81-send-form-label'), input_label_alt=ftl('whatsnew81-send-form-label-alt'), class='vertical') }}
+      </div>
+    {% else %}
+      <div class="qr-code-wrapper">
+        <img src="{{ static('img/firefox/mobile/protocol/qr-firefox.png') }}"
+            width="220"
+            height="220"
+            id="firefox-qr"
+            data-mozillaonline-link="{{ static('img/firefox/mobile/protocol/qr-firefox-mozillaonline.png') }}"
+            alt="{{ ftl('whatsnew81-scan-this-qr-code') }}">
       </div>
+    {% endif %}
     </div>
   </section>
 
-  <section class="content-extra">
+  <section class="wnp-content-extra">
```

Comment:
Note for reviewer: WNP81 only has two picto blocks and they don't feature icons, so to properly test the three-column layout locally you can copy the entire `<section class="content-extra">` from whatsnew-fx80.html and paste it here.

Reply:
Hm actually no, a direct copy-paste won't quite work because a few other things are changing (the icons are a tiny bit bigger and the wrapper needs to be `mzp-t-content-lg`). So... maybe let's just not worry about the three-column layout in this PR and we'll tackle that next time we have a WNP with three blocks.